home/web/client/components/Toasts/Toasts.js

```diff
@@ -33,13 +32,41 @@ export default class Toasts extends Component {
               id={ t.id }
               msg={ t.msg }
               type={ t.type }
-              onRemoveToast={ onRemoveToast }
-              onClickToast={ onClickToast }
+              onRemoveToast={ this.onRemoveToast }
+              onClickToast={ this.onClickToast }
             />
           ))
         }
       </div>
     );
   }
 
+  onClickToast = id => {
+    console.log('onClickToast', id);
+    this.props.onClickToast();
+    this.props.onRemoveToast(id);
```

Comment:
it opens the signer in a new tab and focuses on it
u c any reason to keep the toast open?
(it will b closed after 4 seconds anywayz)